framework/test/mocha/integration/common.js

```diff
@@ -60,6 +60,22 @@ function createBlock(library, transactions, timestamp, keypair, previousBlock) {
 	return block;
 }
 
+function createValidBlockWithSlotOffset(library, transactions, slotOffset, cb) {
```

Comment:
does this have to use callback style?

Reply:
All the functions in this file are following callback approach, and I would like to keep this consistency.